lib/msgr/connection.rb

```diff
@@ -38,12 +37,16 @@ def connect
       connection
     end
 
-    def channel
-      @channel ||= begin
-        channel = connection.create_channel
-        channel.prefetch 1
-        channel
-      end
+    def channel(prefetch = 1)
```

Comment:
Can we have `prefetch` as a keyword argument?

```
def channel(prefetch: 1)
```

This way one does not call channel with an ominous single number.